frontend/yarn.lock

```diff
@@ -1938,16 +1938,11 @@ acorn-walk@^6.0.1:
   resolved "https://registry.yarnpkg.com/acorn-walk/-/acorn-walk-6.2.0.tgz#123cb8f3b84c2171f1f7fb252615b1c78a6b1a8c"
   integrity sha512-7evsyfH1cLOCdAzZAd43Cic04yKydNx0cF+7tiA19p1XnLLPU4dpCQOqpjqwokFe//vS0QqfqqjCS2JkiIs0cA==
 
-acorn@^6.0.1:
+acorn@^6.0.1, acorn@^6.2.1:
```

Comment:
Hmm. This change shouldn't be part of your PR I guess ;-)

Reply:
Ja wie kommt der denn da rein?

Reply:
ach der dependabot